### ISSUE TITLE
Loss Implementation

### ISSUE THREAD

**Issue by brightside51** [brightside51/nc_diffusion#1]
Implementation of **Loss Functions** for Tracking such as **Video Structural Similarity** (_vSSIM_), **Fréchet Inception Distance** (_FID_) and **Inception Score** (_IS_) to the Baseline Model Training Script.
